app/src/main/java/com/example/hikinghelperni/TrailTimeEstimationService.java

```diff
@@ -33,7 +34,8 @@ public ForecastWithHikeTimeSuggestionDTO getUncustomizedDateSuggestionForTrail(T
         LocalDateTime latestHikeTime = getLatestTimeForHike(timeEstimate, forecastDayRecommended);
         TrailHikeTimeSuggestionDTO suggestion =
                 new TrailHikeTimeSuggestionDTO(forecastDayRecommended.getDt(), timeEstimate,
```

Comment:
There is a possibility the timeEstimate could end up a very long double which is quite unnecessary for our needs. However this can be fixed in the next PR